files/securedrop-client

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+mkdir -p ~/.securedrop_client/logs
+mkdir -p ~/.securedrop_client/data
+chmod 0700 ~/.securedrop_client/logs
```

Comment:
As said elsewhere, the Python part of the app should handle all of this. What you're doing here is actually unsafe in the sense that it will force a director with `0777` permissions to `0700` without erroring out. This is def an error, and should not be force corrected by `bash`.

Reply:
so we do have this logic in the app itself - why duplicate it here @kushaldas?